debian/changelog

```diff
@@ -1,5 +1,8 @@
 cloud-init (22.1-14-g2e17a0d6-0ubuntu1~18.04.1) bionic; urgency=medium
 
+  * d/control:
```

Comment:
I think we need a new UNRELEASED changelog entry here because 22.1-14-g2e17a0d6-0ubuntu1~18.04.1 is already officially published. 
So you'll want to `dch -v 22.1-14-g2e17a0d6-0ubuntu1~18.04.4` and put this d/control content into that changelog entry as a separate commit from the d/control changes. This will make it easier to collapse the next new-upstream-snapshot into ubuntu/bionic in the future with changelog commits vs functional commits.

Reply:
`rmadison cloud-init tells` us we already released 18.04.3 so let's bump to the next available version (even though we'll be setting this to something like 22.2  before we officially release next (and syncing in the hotfix d/changelog entries we published from `upstream/ubuntu/bionic-22.1-hotfix`